guides/admin/preconfigured-webhooks/index.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: single
+title:  "Preconfigured Webhook Stages"
```

Comment:
How 'bout "Customer" instead of "Preconfigured"?  It sounds like that's what this feature is--custom webhook stages; am I understanding correctly?

Reply:
My only hesitation is that the feature, in code, is called "preconfigured" and I want to avoid any confusion. I agree that it rolls off the tongue a bit better, though. I'll change it in this PR. We can always change it back if it confuses anyone.